libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -120,12 +120,27 @@ class AC_AttitudeControl {
 
     // get the roll angular velocity limit in radians/s
     float get_ang_vel_roll_max_rads() const { return radians(_ang_vel_roll_max); }
+    // get the roll angular velocity limit in degrees/s
+    float get_ang_vel_roll_max_degs() const { return _ang_vel_roll_max; }
+
+    // set the roll angular velocity limit in degrees/s
+    void set_ang_vel_roll_max_degs(float vel_roll_max) { _ang_vel_roll_max.set(vel_roll_max); }
```

Comment:
I think you need a save call for these.

Reply:
Again these are not tested and is why they are not saved nor does there need to be a save function in attitude control library